formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -38,15 +38,15 @@ class virtual stackWithAttributes 'a init => {
   /* Virtual member */
   val virtual dummy: unit;
   val mutable v: list 'a = init;
-  method virtual implementMe: int => int;
-  method pop =
+  pub virtual implementMe: int => int;
+  pub pop =
```

Comment:
For types yes, but not for methods - I added one but I don't think all the private method cases are covered.